aikau/src/main/resources/alfresco/search/FacetFilter.js

```diff
@@ -187,14 +203,21 @@ define(["dojo/_base/declare",
        *
        * @instance
        */
-      onToggleFilter: function alfresco_search_FacetFilter__onToggleFilter(/*jshint unused:false*/ evt) {
-         if (this.applied)
+      onToggleFilter: function alfresco_search_FacetFilter__onToggleFilter(evt) {
+         if (evt.target !== this.labelNode && !this.fullWidthClick)
```

Comment:
I'm sure this is correct, but just want to be sure this shouldn't be a `||` rather than `&&`?